api/src/api/application_alpha/application_schemas.py

```diff
@@ -12,3 +12,12 @@ class ApplicationStartResponseDataSchema(Schema):
 
 class ApplicationStartResponseSchema(AbstractResponseSchema):
     data = fields.Nested(ApplicationStartResponseDataSchema())
+
+
+class ApplicationFormUpdateRequestSchema(Schema):
+    application_response = fields.Dict(required=True)
+
+
+class ApplicationFormUpdateResponseSchema(WarningMixinSchema):
+    message = fields.String()
+    data = fields.Dict()
```

Comment:
For the response schema, it should also be derived from the AbstractResponseSchema to get the message field, but we should also set data to be equal to something that has the application ID in it.
```suggestion
class ApplicationFormUpdateResponseSchema(AbstractResponseSchema, WarningMixinSchema):
    data = fields.Nested(something)
```